### PR TITLE
Update attacktimer

### PR DESCRIPTION
**Pull request by ngraves95**
* Make timer moveable
* Update icon

### CODE REVIEW

**Comment by Richardant** [comment]
@ngraves95

**Comment by iProdigy** [comment]
@ngraves95 you'll want to merge https://github.com/ngraves95/attacktimer/pull/67 to resolve the build error

**Comment by ngraves95** [comment]
Updated the commit

**Comment by iProdigy** [comment]
One more PR for you https://github.com/ngraves95/attacktimer/pull/69

**Comment by ngraves95** [comment]
Thanks, merged 69 and updated this PR with the correct hash

**Comment by iProdigy** [comment]
apologies, last one https://github.com/ngraves95/attacktimer/pull/70

File: plugins/attacktimer [hunk 1]
```diff
@@ -1,2 +1,2 @@
 repository=https://github.com/ngraves95/attacktimer.git
-commit=a01456308844de3e766ea5fc4d55c6a097bfd92a
+commit=c3f915a69ea757d4c4839aa6ee5a416c165bf888
```
Comment (iProdigy) [inline plugins/attacktimer:2]:
```suggestion
commit=5049ea52853ed4e29fac5d0ec5f47e55be439bb4
```

Comment (Richardant) [reply plugins/attacktimer:2]: @ngraves95

**Review by iProdigy — APPROVED** [review]

**Review by tylerwgrass — APPROVED** [review]

**Pull request merged**